benchmarks/basic_functions.py

```diff
@@ -0,0 +1,148 @@
+"""
+Copyright 2023 Man Group Operations Limited
+
+Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
+
+As of the Change Date specified in that file, in accordance with the Business Source License, use of this software will be governed by the Apache License, version 2.0.
+"""
+from arcticdb import Arctic
+from arcticdb.version_store.library import (
+    WritePayload,
+    ReadRequest
+)
+
+from .common import *
+
+class BasicFunctions:
+    number = 5
+    timeout = 6000
+
+    params = ([50, 100], [500, 1000])
+    param_names = ['rows', 'num_symbols']
```

Comment:
not many rows! Hundreds of thousands would be quite typical

Reply:
I will look into this, more rows means slower benchmarks, so I was conservative to speed-up the iterations.